### PR TITLE
Terms from Issues -> PR

### PR DESCRIPTION
**Pull request by Christian-MK**
Contributors should not use the Issues section. This PR is for clearing out the proposed terms there at present

### CODE REVIEW

**Comment by Christian-MK** [comment]
Terms included in the last commit 
#4, #5, #11, #12, #22, #31, #32, #39

File: content/api.ttl [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+:api
+    skos:prefLabel "API"@en;
+    skos:definition '''
+      A structured and predictable software interface which
+      provides a contract between external software components
+      for accessing data or services and better enabling
+      inter-connectivity between components.
+    ''' ;
+    skos:altLabel "application programming interface ".
```
Comment (waalge) [inline content/api.ttl:11]: Is there any reason we are not just taking a dictionary definition for this? 
image

**Review by waalge — COMMENTED** [review]
I can't sign off on these definitions.

These are general terms for which i am not aware we have any meaningful divergence from how they are generally understood. At worst I think we should cite external sources for them 

API came up over as an issue cos there was disagreement over whether using the blockchain for the interface was an API, since it is more commonly used to refer to HTTP interfaces. (The answer is that it is unambiguously an API as API does not require HTTP.) 
The definition given doesn't resolve the disagreement, and i don't think is nearly as clear as the one provided by a dictionary.

**Comment by ross-spencer** [comment]
Looking at revisiting this tomorrow or Monday depending on today's efforts.

One observation is the commit history here seems to be incorrect and it's not looking trivial to resolve conflicts on the command line. It might be that it was branched from one recently merged? (which is easily done).

I wonder, can you checkout main from fresh, and then add the new `.ttl` files into a new branch created from that? It should make review and merging easier.

**Comment by Christian-MK** [comment]
> One observation is the commit history here seems to be incorrect and it's not looking trivial to resolve conflicts on the command line. It might be that it was branched from one recently merged? (which is easily done).

ah yes, my bad. I'll get this moved over.
And will continue the conversation there

**Comment by Christian-MK** [comment]
A new PR has been made
https://github.com/orcfax/glossary/pull/46

**Pull request closed without merging**